src/components/db.ts

```diff
@@ -10,7 +10,9 @@ export const openDB = async (): Promise<Database> => {
       filename: 'db/bot.db',
       driver: sqlite3.Database
     });
+    //initialize all relevant tables
     await db.run('CREATE TABLE IF NOT EXISTS saved_data (msg_id INTEGER PRIMARY KEY,data TEXT NOT NULL);');
+    await db.run('CREATE TABLE IF NOT EXISTS Interviewers (UserId TEXT, Link TEXT)');
```

Comment:
we should prob make `UserId` the primary key and add not null constraint for `Link`. Also be mindful of the column name styles here, for `saved_data` you are using snake case, but for `Interviewers` you are using camel case. We should just stick with one, personally, I've always done snake case for SQL stuff. So `interviewers` table with columns `user_id` and `link`

Reply:
Should I use a String as primary key though? I was worried about performance issues. Fixing everything else in a bit.

Reply:
I think having a primary key/index is better than not having one. If you are concerned about integer comparisons vs string comparisons, I think it's fine because we won't have a lot of rows, and the fact that we don't have to maintain our own integer primary key as well as discord user ID foreign key outweighs the slight performance decrease.

Reply:
Actually isn’t discord IDs numeral objects? I think we can use integers in sqlite3 then right?

Reply:
ID's are too big for Javascript to hold them precisely apparently. When I had them as numbers the last few digits had precision issues.

Reply:
oh ok then using text is fine